### DISCUSSION TITLE
请问5月底是否有计划发布新的v5稳定版本?

### DISCUSSION THREAD

**Discussion by ChristmasFox** [antvis/G2#5067] [General]
请问5月底是否有计划发布新的v5稳定版本?

**Comment by hustcc** [comment]
当前已经发布正式版本，是遇到什么问题吗？可以反馈出来我们，我们紧急跟进修复。5 ~ 6 两个月主要内容：

- auto padding：https://github.com/antvis/G2/pull/5046 ✅
- 自定义能力走查
- 新主题和主题工具
- 官网文档、案例走查

**Reply by ChristmasFox** [reply]
您好 是的 我们现在遇到一些自定义相关的问题。

1. 在图例交互中 legend-visible-filter 这个方法不再v5的计划中吗？我们想让点击图例对应的图形隐藏/显示，不会引起坐标轴的变化。
2. registerShape 方法是否被删除了？我们需要在v5中也能够自定义图形，v5中是否有类似draw(cfg: ShapeInfo, container: IGroup)这个方法能够使我获取相应标记的坐标点，来绘制自定义的图表。

**Reply by hustcc** [reply]
相关问题：

- https://github.com/antvis/G2/discussions/5071
- https://github.com/antvis/G2/discussions/5059

看到你提的讨论还是挺多的，也可以加我们群，虽然也不是实时回答，但是可以加强联系，多给我们群里提建议。
